### DISCUSSION TITLE
how to order the pages in descending order in nav bar?

### DISCUSSION THREAD

**Discussion by jackywu** [web-infra-dev/rspress#1657] [Q&A]
I saw we could have the pages in ascending order by prefix them with number.
![image](https://github.com/user-attachments/assets/3fbf4925-d2c8-482a-9788-86cd9013d148)

but on my site, I have date string at front of the file name, and it also make all the pages order in ascending. 
![image](https://github.com/user-attachments/assets/0e09fb41-c60f-4d63-9d20-720322c75b28)

but I want them ordered in descending order. how to do this ? thanks.

**Comment by Timeless0911** [comment]
Using `_meta.json`.
